server/routes/visit/visitUtils.ts

```diff
@@ -129,6 +146,21 @@ export const getVisitNotificationsAlerts = (notifications: VisitBookingDetails['
   return alerts
 }
 
+export const getVisitAlerts = (visitDetails: VisitBookingDetails): MoJAlert[] => {
```

Comment:
Just combining the output of the three functions above, which are no longer exported.